managed-services/transactional-email/how-to/add-dmarc-record.mdx

```diff
@@ -0,0 +1,51 @@
+---
+meta:
+  title: How to configure a DMARC record
+  description: This page explains how to configure a DMARC record
+content:
+  h1: How to configure a DMARC record
+  paragraph: This page explains how to configure a DMARC record
+tags: spf spf-record dkim dkim-records dmarc dmarc-record transactional
+dates:
+  validation: 2024-02-21
+  posted: 2024-02-21
+categories:
+  - managed-services
+---
+
+This page shows you how to configure a basic [DMARC record](/managed-services/transactional-email/concepts/#dmarc-record) on your domain from the **Scaleway console**.
+If you have registered a domain at another provider, you can still use this documentation, as the process is similar.
+For more information about DMARC configuration and examples of DMARC records you can configure, refer to the [dedicated documentation](/managed-services/transactional-email/reference-content/dmarc-configuration/).
+
+## Requirements
+
+- A Scaleway account logged into the [console](https://console.scaleway.com/).
+- Have a domain name.
+- Have [configured your domain with Transactional Email](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email).
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+
+## How to configure a DMARC record
+
+1. Click **Transactional Email** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu.
+2. Click your domain. You are redirected to the records configuration page.
+3. Add a `TXT` record type in the same zone as your Transactional Email's domain, on your DNS zone.
+4. Enter `_dmarc` in the **Name** field, followed by the domain name. For example, if your domain is called `my-domain.com`, you should have `_dmarc.my-domain.com.` in the **Name** field.
+5. Enter your desired tags and their values in the **Value** section. Make sure that you add the mandatory `v` and `p` tags, and that you separate tags from each other with semi-colons. Find out about DMARC configuration in our [dedicated documentation](/managed-services/transactional-email/reference-content/dmarc-configuration/).
+    Your **Value** field should look similar to the following
+    ```
+    v=DMARC1; p=none; rua=mailto:your@email.com
+    ```
+    <Message type="important">
+     It is possible to configure several DMARC records for one domain, but they must have a different name to be valid.
+     For example, the `_dmarc.mydomain.com` and `_dmarc.tem.mydomain.com` DMARC records are valid.
+    </Message>
+6. Click **Add record**.
+
+<Message type="important">
+  Emails sent from sources not configured in your SPF record, or that do not have a DKIM **will be rejected if you use the `p=reject` policy**.
+</Message>
```

Comment:
I don't actually get why we add this note here, as we don't talk about "p=reject" before.
I would rather see it on the "Understanding DMARC configuration" (where is already is :) )
Or let it here but make it in accordance with the example record " v=DMARC1; p=none; rua=mailto:your@email.com" mentioned above.